pkg/commands/git_commands/rebase.go

```diff
@@ -423,23 +423,25 @@ func (self *RebaseCommands) runSkipEditorCommand(cmdObj oscommands.ICmdObj) erro
 }
 
 // DiscardOldFileChanges discards changes to a file from an old commit
-func (self *RebaseCommands) DiscardOldFileChanges(commits []*models.Commit, commitIndex int, fileName string) error {
+func (self *RebaseCommands) DiscardOldFileChanges(commits []*models.Commit, commitIndex int, filePaths []string) error {
```

Comment:
I spent some time with the [git book](https://git-scm.com/book/en/v2) and got a much better understanding of what's happening here. I've updated this in line with https://github.com/jesseduffield/lazygit/pull/3276#discussion_r1469080802.